monitoring/exposing-custom-application-metrics-for-autoscaling.adoc

```diff
@@ -9,3 +9,7 @@ You can export custom application metrics for the horizontal pod autoscaler.
 
 include::modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc[leveloffset=+1]
 
+.Additional resources
+
+See the xref:../nodes/pods/nodes-pods-autoscaling.adoc#nodes-pods-autoscaling[horizontal pod autoscaling documentation].
```

Comment:
Could add a bullet to the front of this, but also I see in the built version of the guide, there's an `Additional resources` already at the end of the `monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling` module, which appears right before this:

```
For information about Horizontal Pod Autoscaler, see link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[the Kubernetes documentation].
```

So the `Additional resources` are stacking. Maybe move the kube doc link to this assembly instead and have 2 bullets?

Reply:
Good catch, done.